frontend/app/src/composables/blockchain/index.ts

```diff
@@ -140,16 +140,18 @@ export function useBlockchains() {
         if (added) {
           const [address, chains] = Object.entries(added)[0];
 
-          chains.forEach((chain) => {
+          const isAll = chains.length === 1 && chains[0] === 'all';
+
+          const usedChains = isAll ? get(evmChains) : chains;
```

Comment:
`evmChains` here also contains `AVAX` right, is `AVAX` part of this addition endpoint?